packages/input_secrets/package.json

```diff
@@ -0,0 +1,46 @@
+{
+    "name": "@apify/input_secrets",
```

Comment:
why underscore in the name?

Reply:
It needs to be same as folder path because https://github.com/apify/apify-shared-js/blob/master/tsconfig.json#L6

Reply:
ok, understandable :D still it's common to have a dash instead of underscore